src/Repository/Parser.cpp

```diff
@@ -24,9 +24,16 @@ Copyright_License {
 #include "Parser.hpp"
 #include "FileRepository.hpp"
 #include "IO/LineReader.hpp"
+#include "IO/FileLineReader.hpp"
 #include "Util/StringStrip.hxx"
 #include "Util/HexString.hpp"
+#include "LocalPath.hpp"
 
+/**
+ * Parses a line of the repository file.
+ * Each line is of the form `name = value`
+ * @returns A pointer to the value field.
+ */
```

Comment:
Adding documentation is always good, but please make this a separate commit. I'd like to cherry-pick this change, but I can't.

Reply:
Split out.